### PR TITLE
Allow PercyAgent to be exported onto the window as PercyAgent

### PR DESCRIPTION
**Pull request by djones**
screen shot 2018-09-12 at 10 30 45 am

### CODE REVIEW

**Comment by djones** [comment]
This comment was helpful at figuring this out: https://github.com/TypeStrong/tsify/issues/105#issuecomment-241936617

File: src/percy-agent-client/percy-agent-service.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+export class PercyAgentService {
```
Comment (fotinakis) [inline src/percy-agent-client/percy-agent-service.ts]: This is more like a `PercyAgentClient`, right?

File: src/percy-agent-client/percy-agent.ts [hunk 2]
```diff
@@ -37,7 +19,7 @@ export class PercyAgentClient {
   snapshot(name: string, options: SnapshotOptions = {}) {
     let documentObject = options.document || document
     let domSnapshot = this.domSnapshot(documentObject)
-    let percyAgent = new PercyAgent(this.xhr)
+    let percyAgent = new PercyAgentService(this.xhr)
```
Comment (fotinakis) [inline src/percy-agent-client/percy-agent.ts]: Maybe `percyAgentService`?

File: src/percy-agent-client/percy-agent-service.ts (hunk 1 above)
Comment (fotinakis) [reply src/percy-agent-client/percy-agent-service.ts]: He says, non-ironically.

**Pull request merged**